lib/aggregation/accumulator/package.json

```diff
@@ -43,7 +43,6 @@
     "abacus-rating-config": "file:../../config/rating",
     "abacus-router": "file:../../utils/router",
     "abacus-request": "file:../../utils/request",
-    "abacus-seqid": "file:../../utils/seqid",
```

Comment:
With the revert of the refactoring you did, the `seqid` is now again required by the `accumulator` module.